helpers/integration.go

```diff
@@ -356,6 +356,25 @@ func GetAllChildTaskRunsForPipelineRun(adapterClient client.Client, ctx context.
 	return taskRuns, nil
 }
 
+// GetAllTaskRunsWithMatchingPipelineLabel finds all Child TaskRuns
+// whose "tekton.dev/pipeline" label points to the given PipelineRun
+func GetAllTaskRunsWithMatchingPipelineLabel(adapterClient client.Client, ctx context.Context, pipelineRun *tektonv1.PipelineRun) (*tektonv1.TaskRunList, error) {
```

Comment:
I think this function would be better placed in the [loader](https://github.com/redhat-appstudio/integration-service/blob/main/loader/loader.go) package.

Reply:
Thanks for letting me know, I've made the required changes for this move.

Reply:
Oops, done! PTAL